jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/ClientUpgradeRequest.java

```diff
@@ -161,13 +161,13 @@ public String getHost()
     @Override
     public String getHttpVersion()
     {
-        return httpVersion;
+        throw new UnsupportedOperationException("HttpVersion not available on ClientUpgradeRequest");
     }
 
     @Override
     public String getMethod()
     {
-        return method;
+        throw new UnsupportedOperationException("Method not available on ClientUpgradeRequest");
```

Comment:
Return hardcoded `HttpMethod.GET.asString()`.

Reply:
Should just remove the `.getMethod()` entirely, it's always 'GET'

Reply:
Same as above, for HTTP/2 this would be `CONNECT`. So at this stage we don't really know.